tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -229,22 +230,36 @@ def initial_mirror_config(self, mirror1, mirror2, poolname, imagename, **kw):
             mode - mirror mode to be configured pool or image
             mirrormode - type of mirror configured journal or snapshot
         """
-        mode = kw.get("mode")
         imagespec = poolname + "/" + imagename
         imagesize = kw.get("imagesize")
         io = kw.get("io_total")
-        mirror1.create_pool(poolname=poolname)
```

Comment:
[optional][needhelp] [Just to know]
Why can't we access mirror1 methods and variables using `self` instead of duplicate local object `mirror1` ?
I see that having new object makes it easier to read and understand, but how to look at it from programing best practices ?
cc @manasagowri

Reply:
i think from understanding purpose, it is good to have local object, if you see in code we have to use same methods on both the object mirror1 and mirror2.

Reply:
I agree, we can just use self, instead of again passing another parameter as mirror1, i think even from programming perspective that looks more pythonic.

Reply:
Done.